docker/gaffer-kerberos/gaffer-integration-tests-krb/krb-entrypoint.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Copyright 2020-2023 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+# Show verbose Hadoop Kerberos auth information
+if [ "$DEBUG" -eq 1 ]; then
+  export HADOOP_JAAS_DEBUG=true
+  export HADOOP_OPTS="-Djava.net.preferIPv4Stack=true -Dsun.security.krb5.debug=true -Dsun.security.spnego.debug"
+  echo "Debugging flag enabled (DEBUG=$DEBUG), additional Kerberos details will be printed"
+fi
+
+KEYTAB_PATH=/tmp/gaffer.keytab
+PRINCIPAL=gaffer/$(hostname)
+FULL_PRINCIPAL=gaffer/$(hostname)@GAFFER.DOCKER
+
+{
+echo "add_entry -password -p $PRINCIPAL -k 1 -e aes256-cts"; sleep 0.2
+echo $GAFFER_KRB_PASSWORD; sleep 0.2
+echo list; sleep 0.2
+echo "write_kt $KEYTAB_PATH"; sleep 0.2
+echo exit
+} | ktutil
+
+# Update store properties files to point to the location of the Accumulo store to test against:
+accumulo_instance=$(cat conf/store.properties | grep accumulo.instance | sed -e 's/.*=\(.*\)/\1/')
+accumulo_zookeepers=$(cat conf/store.properties | grep accumulo.zookeepers | sed -e 's/.*=\(.*\)/\1/')
+store_properties=$(find /tmp/gaffer/store-implementation/accumulo-store/src/test/resources -name *.properties | grep -v cache)
```

Comment:
Is this meant to be an array or just first match? If its meant to be an array then it should be wrapped in outer brackets e.g.
```sh
declare -a ($(find ... | grep ...))
```